### ISSUE TITLE
Token lines displaying on a single line

### ISSUE THREAD

**Issue by jonmilner** [FormidableLabs/react-live#129]
Expected:

<img width="1008" alt="Screen Shot 2019-04-01 at 12 42 43 PM" src="https://user-images.githubusercontent.com/937847/55344571-ae178b80-547b-11e9-8539-431cb4d19711.png">

Actual:

<img width="1008" alt="Screen Shot 2019-04-01 at 12 05 32 PM" src="https://user-images.githubusercontent.com/937847/55344580-b4a60300-547b-11e9-8dc2-474d25243a47.png">

This is what I'm seeing in a brand new create-react-app repo (https://github.com/jonmilner/react-live) with the following code:

```
import * as React from "react";
import { render } from "react-dom";

import { LiveProvider, LiveEditor, LiveError, LivePreview } from "react-live";

const code = `<div>
  <div>
    <h1>Hello World</h1>
  </div>
</div>`;

function App() {
  return (
    <LiveProvider code={code}>
      <LivePreview />
      <LiveError />
      <LiveEditor />
    </LiveProvider>
  );
}

const rootElement = document.getElementById("root");
render(<App />, rootElement);
```

Interestingly, removing `display: inline` from the selector `.npm__react-simple-code-editor__container > pre *` in the browser dev tools fixes the issue. However, it doesn't seem it can be fixed by overriding the style with any valid property.

<img width="1008" alt="Screen Shot 2019-04-01 at 12 47 20 PM" src="https://user-images.githubusercontent.com/937847/55344880-8ffe5b00-547c-11e9-95e8-9ff85d44ff72.png">

I'm seeing this in Chrome (v73.0.3683.86) on macOS (10.14.3)

**Comment by wangdicoder** [comment]
I met this problem as well.

**Comment by feychenie** [comment]
same

**Comment by feychenie** [comment]
This breaking CSS rule is added in the latest update on react-simple-code-editor:

See this commit:

https://github.com/satya164/react-simple-code-editor/commit/6feba9690d32238bb0e2e17593cde5e17be5c67d

**Comment by sofiapoh** [comment]
This should be fixed by https://github.com/satya164/react-simple-code-editor/commit/a704a08c8a1727297aff2e32f70dfe97db1173c0, let me know if the problem persists!

**Comment by feychenie** [comment]
Yes fixed in react-simple-code-editor@0.9.10
